src/main/scala/org/broadinstitute/hail/driver/ImportPlink.scala

```diff
@@ -30,7 +30,7 @@ object ImportPlink extends Command {
 
     @Args4jOption(required = false, name = "-d", aliases = Array("--delimiter"),
       usage = ".fam file field delimiter regex")
-    var famDelimiter: String = "\\t"
+    var famDelimiter: String = "\\\\s+"
```

Comment:
It looks like Plink 1.07 allows space or tab delimited, and Plink 1.9 is flexible while transitioning to Plink 2.00 which will be tab delimited.  I agree with Tim that IDs with spaces should not be allowed for Plink import:
https://www.cog-genomics.org/plink2/filter#indiv

Tim, I'd say just make sure there's a useful error message when an id does have a space.